src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/index.tsx

```diff
@@ -134,14 +134,16 @@ const SmallCard: React.FC<React.PropsWithChildren<IfoCardProps>> = ({
     )
   }, [account, ifo, poolId, publicIfoData, vestingInformation, walletIfoData])
 
+  const cardTitle = ifo.isCommunity ? `${config.title} (cIFO)` : config.title
```

Comment:
Hmm. Currently there is a logic if point is required and squad isn't, it's cIFO. We should either choose the current one or replace with the flag here for other place to avoid multiple logic for the same state.